src/ui/common/src/stories/CodeBlock.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React from 'react';
+
+import CodeBlock from '../components/CodeBlock';
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'Components/Code Block',
+  component: CodeBlock,
+  // More on argTypes: https://storybook.js.org/docs/react/api/argtypes
+  argTypes: {},
+} as ComponentMeta<typeof CodeBlock>;
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Template: ComponentStory<typeof CodeBlock> = (args) => (
+  <CodeBlock {...args} />
+);
+
+const apiConnectionSnippet = `import aqueduct
+client = aqueduct.Client(
+    "QYIF0375KE4Z2GXSB6T8RNJLVMA9WCPH",
```

Comment:
Do we need to change this to something like `YOUR_API_KEY` ?

Reply:
Yes, I think that's a better idea than just using the API key that I have right now. Going to make this change.